src/vm/exceptionhandling.cpp

```diff
@@ -3056,7 +3056,19 @@ CLRUnwindStatus ExceptionTracker::ProcessManagedCallFrame(
                             }
                             else
                             {
-                                TypeHandle typeHnd = pJitMan->ResolveEHClause(&EHClause, pcfThisFrame);
+                                TypeHandle typeHnd;
+                                EX_TRY
+                                {
+                                    typeHnd = pJitMan->ResolveEHClause(&EHClause, pcfThisFrame);
+                                }
+                                EX_CATCH_EX(Exception)
```

Comment:
What was the behavior on Windows when the exception escaped here? Was it crashing like on Unix; or was the exception thrown gracefully?

I am just wondering whether this change is a potential regression on Windows.